distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -33,6 +33,7 @@
         <exclude>etc/org.apache.karaf.log.cfg</exclude>
         <exclude>etc/org.ops4j.pax.web.cfg</exclude>
         <exclude>etc/users.properties</exclude>
+        <exclude>etc/startup.properties</exclude>
```

Comment:
I contemplated that, but I'd have to leave that comment in several files. I'm not opposed, but it's a lot of noise.